### PR TITLE
Serializer pass

### PR DESCRIPTION
**Pull request by willy-ahva**
| Q                    | A
| -------------------- | ---
| Bug fix?             | n
| New feature?         | n
| BC breaks?           | n
| CI currently passes? | y
| Tests pass?          | y
| Scenarios pass?      | y
| Checkstyle issues?*  | n
| PMD issues?**        | n
| Changelog updated?   | n
| Fixed tickets        | PIM-3292
| DB schema updated?   | n
| Migration script?    | n
| Doc PR               | n

### CODE REVIEW

File: src/Pim/Bundle/TransformBundle/DependencyInjection/Compiler/SerializerPass.php [hunk 1]
```diff
@@ -55,6 +55,13 @@ public function process(ContainerBuilder $container)
         $encoderTag = sprintf("%s.encoder", $this->serializerServiceId);
         $encoders = $this->findAndSortTaggedServices($encoderTag, $container);
 
+        if (empty($normalizers) && empty($encoders)) {
+            throw new \RuntimeException(
+                sprintf('You must tag at least one normalizer or encoder as "%s" to use the Serializer service',
+                    $this->serializerServiceId)
```
Comment (sumbobyboys) [inline src/Pim/Bundle/TransformBundle/DependencyInjection/Compiler/SerializerPass.php]: bad indentation

File: src/Pim/Bundle/TransformBundle/DependencyInjection/Compiler/SerializerPass.php [hunk 2]
```diff
@@ -55,6 +55,13 @@ public function process(ContainerBuilder $container)
         $encoderTag = sprintf("%s.encoder", $this->serializerServiceId);
         $encoders = $this->findAndSortTaggedServices($encoderTag, $container);
 
+        if (empty($normalizers) && empty($encoders)) {
+            throw new \RuntimeException(
```
Comment (sumbobyboys) [inline src/Pim/Bundle/TransformBundle/DependencyInjection/Compiler/SerializerPass.php]: do the symfony one act like this ?

File: spec/Pim/Bundle/TransformBundle/DependencyInjection/Compiler/SerializerPassSpec.php [hunk 3]
```diff
@@ -80,6 +80,41 @@ function it_sets_arguments_of_pim_serializer_with_tagged_normalizers_and_denorma
         $this->process($container);
     }
 
+    function it_sets_arguments_of_pim_serializer_with_tagged_normalizers_and_no_encoders_by_default(
```
Comment (sumbobyboys) [inline spec/Pim/Bundle/TransformBundle/DependencyInjection/Compiler/SerializerPassSpec.php]: don't understand the relation between your pr and this spec

Comment (willy-ahva) [inline spec/Pim/Bundle/TransformBundle/DependencyInjection/Compiler/SerializerPassSpec.php]: it checks there is no exception with a normalizer and no encoder. That's the point a modified with this PR. It allows to have tagged normalizers without encoders. Before you needed encoders.

**Pull request closed without merging**